components/brave_wallet_ui/common/hardware/ledgerjs/ledger-trusted-transport.test.ts

```diff
@@ -20,12 +20,7 @@ const createWindow = (): Window => {
   let iframe = document.createElement('iframe')
   document.body.appendChild(iframe)
   if (!iframe.contentWindow) { fail('transport should be defined') }
-  // Use Object.defineProperty in order to assign to
-  // window.crypto because standard assignment results in
-  // assignment error because window.origin is read-only
-  Object.defineProperty(iframe.contentWindow, 'origin', {
-    value: 'chrome-untrusted://ledger-bridge'
-  })
```

Comment:
I'm not sure this one is actually causing the error (although I don't see why it wouldn't either) - the issues are all complaining about window.crypto being read-only.  I think the error appears in three files because `MockLedgerTransport` is imported into two additional files.

Reply:
I also opened this one to fix https://github.com/brave/brave-core/pull/14746